guides/common/modules/ref_supported-clients.adoc

```diff
@@ -0,0 +1,61 @@
+[id="supported-clients_{context}"]
+= Supported Clients
+
+{Project} supports the following operating systems and architectures.
+
+Supported Host Operating Systems::
+The hosts can use the following OS versions:
+
+ifdef::orcharhino[]
+* Alma Linux
+* Amazon Linux
+endif::[]
+ifndef::satellite[]
+* CentOS
+endif::[]
+ifndef::orcharhino,satellite[]
+* CoreOS
+endif::[]
+ifndef::satellite[]
+* Debian
+ifndef::orcharhino[]
+* Fedora
+* FreeBSD
+* Junos
+* OpenSUSE
+endif::[]
+* Oracle Linux
+endif::[]
+* {EL}
+** 9.0 or later
+** 8.0 or later
```

Comment:
Isn't it sufficient to specify only the major if all versions are supported? `or later` could also be interpreted that it includes EL 10 (once it's released).
```suggestion
** 9
** 8
```